client/src-ssr/index.js

```diff
@@ -11,37 +11,36 @@
  *   "src-ssr/extension.js"
  */
 
-const
-  express = require('express'),
-  compression = require('compression')
+const express = require("express"),
+  compression = require("compression");
```

Comment:
I'm fine with keeping these added semicolons for now, but I prefer JS without semicolons these days. 

We should probably formally establish our code style preferences as a team at some point.

Reply:
Yeah that's next on my list. These changes were added by vscode's format on save feature.